console/console-init/ui/src/stories/DeviceList.stories.tsx

```diff
@@ -20,7 +20,7 @@ import { IRowData } from "@patternfly/react-table";
 import { DeviceDetailNavigation } from "modules/device-detail";
 
 export default {
-  title: "Device"
+  title: "Device List Table"
```

Comment:
The stories in this file are not limited to `Device List Table`. It includes everything in the device page.

Reply:
Stories doesn't allow to export same title from two files.